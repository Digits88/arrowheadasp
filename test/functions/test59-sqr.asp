<!-- Test Sqr function.
-->

Sqr(0) = <%= Sqr(0) %><br>
Sqr(1) = <%= Sqr(1) %><br>
Sqr(2) = <%= Sqr(2) %><br>
Sqr(3) = <%= Sqr(3) %><br>
Sqr(4) = <%= Sqr(4) %><br>
