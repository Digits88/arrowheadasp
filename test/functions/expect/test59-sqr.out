<!-- Test Sqr function.
-->

Sqr(0) = 0<br>
Sqr(1) = 1<br>
Sqr(2) = 1.41421356^[0-9]*$<br>
Sqr(3) = 1.73205080^[0-9]*$<br>
Sqr(4) = 2<br>
